swift/ios-shoppe-demo/Model/Product.swift

```diff
@@ -12,7 +12,8 @@ import UIKit
 class Product {
     var title: String
     var description: String
-    var price: Decimal
+    var rawPrice: Double
```

Comment:
I think I liked what you had the first time.  `price` should indicate the actual numeric price.  If you're trying to show some intermediate representation to the user, then you typically see `displayPrice` or something.  But here is where I'd say if we're dealing with formatting, that's not the job of a Model - that's the job of a View.  So you may consider moving anything that deals with representing the price to a user (i.e. it has a `$` before it) to elsewhere in code.